README.md

```diff
@@ -79,6 +117,38 @@ EXAMPLES
 
 _See code: [dist/commands/append/fact.ts](https://github.com/evently-cloud/cli/blob/v0.2.0/dist/commands/append/fact.ts)_
 
+## `evently append:serial`
+
+Appends a serial event to the ledger
+
+```
+USAGE
+  $ evently append:serial [-t <value>] [-f <value>] [-b <value>]
+
+FLAGS
+  -b, --body=<value>   Read data from argument. If neither --body nor --file is given, STDIN is used.
+  -f, --file=<value>   Read event data from this file. If neither --body nor --file is given, STDIN is used.
+  -t, --token=<value>  [default: NOT-SET] Access token for your ledger.
+
+DESCRIPTION
+  Appends a serial event to the ledger
+
+EXAMPLES
+  $ evently append:serial <<EVENT
+  {
+    entity: "thermostat",
+    event: "temperature-recorded",
+    key: "thermostat1",
+    meta: {causation: "14rew3494"},
+    data:{celsius: 18.5},
+    previousEventId: "foo-bar"
```

Comment:
Nice, I had this as a param, but it makes sense in the CLI body too. Helps the dev understand that `previousEventId` is part of the request body rather than a query param or something else.